docs/UserGuide.md

```diff
@@ -111,20 +111,23 @@ Examples:
 
 ### Locating persons by criteria: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons whose fields contain the given keywords.
```

Comment:
Maybe can also update the command summary table at the bottom of the file?

Reply:
Good catch, I will update accordingly. Thank you.